### PR TITLE
Document OpenMRS Path app

### PR DESCRIPTION
**Pull request by ekigamba**
Add screenshots about the app, brief app walkthrough for end-user & explanations of widgets, colors & abbreviations used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,298 @@
...
+   * [Stock Control](#5-slide-menu)
+   * [HIA 2 Reports](#5-slide-menu)
+* [Project Dependencies](#project-dependencies)
+   * [Folder Structure](#folder-structure)
 
-1. Set the `ANDROID_HOME` environment variable to point to the location of your installed Android SDK 4.1.2 API level 16. For more information, look at [the documentation of maven-android-plugin](http://code.google.com/p/maven-android-plugin/wiki/GettingStarted).
 
-2. Start an Android Virtual Device. Normally, this means you need to run `android avd` and then start one of the devices there.
+# Introduction
+The OpenSRP Path is an app that handles infant records from child birth to 5 years of age. This mainly includes monitoring child weight, vaccination schedules and other supplements that have been proven to prevent diseases and/or, improve infant growth and health.
```
Comment (pld) [inline README.md]: remove first "The"

Comment (pld) [reply README.md]: remove comma after "and/or"

File: README.md [hunk 2]
```diff
@@ -3,27 +3,298 @@
...
 
-Then, you can run `mvn clean install` in the main directory.
+It's main focus is **Infant Health**
```
Comment (pld) [inline README.md]: Remove possessive apostrophe, this is not a possessive

File: README.md [hunk 3]
```diff
@@ -3,27 +3,298 @@
...
+   _All the plugins required are explicitly stated therefore can work with any Android Studio version - Just enable it to download any packages not available offline_
+1. Open Genymotion and Run the Virtual Device created previously.
+1. Run the app on Android Studio and chose the Genymotion Emulator
+
+
+
+
+## App Walkthrough (End-User)
+
+1. **(Optional)** Open the app, open the app menu > Settings > Change url to ```http://46.101.51.199:8080/opensrp```
```
Comment (pld) [inline README.md]: should we note that this webserver may not be up? i.e. we don't want to be responsible for keeping this up forever

File: README.md [hunk 4]
```diff
@@ -3,27 +3,298 @@
...
+
+### 4.1 Click on a patient
+
+This opens a page which contains the immunisation history in a vaccine card.
+
+##### Other details
++ Patient name
++ Patient Weight _(Two Options Available: Record past OR Current weight)_
++ Patient Weight Graph
++ Patients Date of Birth - Written as ` DOB: 30/04/2016 `
```
Comment (pld) [inline README.md]: add possessive apostrophe

File: README.md [hunk 5]
```diff
@@ -3,27 +3,298 @@
...
+
+
+As shown below:
+
+![Patient Details](https://user-images.githubusercontent.com/31766075/30325644-36e0fae4-97ce-11e7-8045-f2bc5236d42c.png)
+
+
+### 4.2 Click on the Health Facility Name
+
+A dropdown menu appears with a list of the Health Facility Zones + Health Facility. The healt facility name is the first, the rest are branches/zones of the health facility.
```
Comment (pld) [inline README.md]: missing "h" at the end of "healt facility"

File: README.md [hunk 6]
```diff
@@ -3,27 +3,298 @@
...
+
+![Health Facility Dropdown Menu](https://user-images.githubusercontent.com/31766075/30325653-37290410-97ce-11e7-85b2-16346528a11a.png)
+
+When chosen:
+> - A Health Facility - A list of patients in that health facility from the various health zones will be shown. Arranged in descending order with the latest patient to be served at the to.
+> - A Health Facility Zone - A list of patients in the specific Health Facility zone(Branch of the Health Facility) is shown. Arranged in descending order with the latest patient to be served at the to.
+
+### 4.3 Click on the Plus Icon
+
+An form to add a new patient to the current chosen Health Facility OR Health Facility Zone is openned.
```
Comment (pld) [inline README.md]: "An" => "A"

File: README.md [hunk 7]
```diff
@@ -3,27 +3,298 @@
...
+   CNE | Child Not Exposed
+
+
+### 4.4 Click on the search icon
+
+This opens another activity where the provider can perform an [Advanced Search](#advanced-search) or [Global Search](#global-search) of a patient.
+
+#### Advanced Search
+
+This patient search enabled one to search using other conditions or characteristics.
```
Comment (pld) [inline README.md]: "enabled" => "enables"

File: README.md (hunk 3 above)
Comment (ekigamba) [reply README.md]: Noted

**Pull request merged**